mlflow/protos/databricks_uc_registry_messages.proto

```diff
@@ -328,5 +331,5 @@ message FinalizeModelVersionRequest {
 
 
 message FinalizeModelVersionResponse {
+  optional ModelVersion model_version = 1;
```

Comment:
Included here to avoid a redundant re-fetch of the model version to return to the user, after it's created & finalized